src/main/properties/datasets.properties

```diff
@@ -319,12 +319,37 @@ org.aksw.gerbil.datasets.definition.IndEL-Spe.cacheable=true
 org.aksw.gerbil.datasets.definition.IndEL-Spe.experimentType=A2KB
 
 ### Kore50
-org.aksw.gerbil.datasets.KORE50.file=${org.aksw.gerbil.DataPath}/datasets/KORE50/kore50-nif.ttl
-org.aksw.gerbil.datasets.definition.KORE50.name=KORE50
-org.aksw.gerbil.datasets.definition.KORE50.class=org.aksw.gerbil.dataset.impl.nif.FileBasedNIFDataset
-org.aksw.gerbil.datasets.definition.KORE50.cacheable=true
-org.aksw.gerbil.datasets.definition.KORE50.experimentType=A2KB
-org.aksw.gerbil.datasets.definition.KORE50.constructorArgs=${org.aksw.gerbil.datasets.KORE50.file},${org.aksw.gerbil.datasets.definition.KORE50.name}
+#Below commented dataset is outdated, added Kore50 DBPedia instead of it
+#org.aksw.gerbil.datasets.KORE50.file=${org.aksw.gerbil.DataPath}/datasets/KORE50/kore50-nif.ttl
+#org.aksw.gerbil.datasets.definition.KORE50.name=KORE50
+#org.aksw.gerbil.datasets.definition.KORE50.class=org.aksw.gerbil.dataset.impl.nif.FileBasedNIFDataset
+#org.aksw.gerbil.datasets.definition.KORE50.cacheable=true
+#org.aksw.gerbil.datasets.definition.KORE50.experimentType=A2KB
+#org.aksw.gerbil.datasets.definition.KORE50.constructorArgs=${org.aksw.gerbil.datasets.KORE50.file},${org.aksw.gerbil.datasets.definition.KORE50.name}
+org.aksw.gerbil.datasets.KORE50_YAGO.file=${org.aksw.gerbil.DataPath}/datasets/KORE50/DYWC/KORE_50_YAGO.ttl
+org.aksw.gerbil.datasets.definition.KORE50_YAGO.name=KORE50 YAGO
+org.aksw.gerbil.datasets.definition.KORE50_YAGO.class=org.aksw.gerbil.dataset.impl.nif.FileBasedNIFDataset
+org.aksw.gerbil.datasets.definition.KORE50_YAGO.cacheable=true
+org.aksw.gerbil.datasets.definition.KORE50_YAGO.experimentType=A2KB
+org.aksw.gerbil.datasets.definition.KORE50_YAGO.constructorArgs=${org.aksw.gerbil.datasets.KORE50.file},${org.aksw.gerbil.datasets.definition.KORE50.name}
+org.aksw.gerbil.datasets.KORE50_Wiki.file=${org.aksw.gerbil.DataPath}/datasets/KORE50/DYWC/KORE_50_Wikidata.ttl
+org.aksw.gerbil.datasets.definition.KORE50_Wiki.name=KORE50 Wiki
+org.aksw.gerbil.datasets.definition.KORE50_Wiki.class=org.aksw.gerbil.dataset.impl.nif.FileBasedNIFDataset
+org.aksw.gerbil.datasets.definition.KORE50_Wiki.cacheable=true
+org.aksw.gerbil.datasets.definition.KORE50_Wiki.experimentType=A2KB
+org.aksw.gerbil.datasets.definition.KORE50_Wiki.constructorArgs=${org.aksw.gerbil.datasets.KORE50.file},${org.aksw.gerbil.datasets.definition.KORE50.name}
```

Comment:
Won't work, because the keys on the right side are not defined.

Reply:
rectified keys